types/node/events.d.ts

```diff
@@ -585,16 +598,40 @@ declare module "events" {
              */
             readonly asyncResource: EventEmitterReferencingAsyncResource;
         }
+
+        /** The events always emitted by EventEmitter itself */
+        export interface EventEmitterBuiltInEventMap {
+            newListener: [eventName: string | symbol, listener: Function];
+            removeListener: [eventName: string | symbol, listener: Function];
+        }
+
+        /** Helper type for building functions that take a listener as a paramater when working with event maps */
+        export type EventEmitterEventMapListener<Events extends EventMap<Events>, EventName> = Listener<
+            Events,
+            EventName
+        >;
```

Comment:
What sort of usage case did you have in mind? The test example you mentioned seems quite niche...!